File: september/23-09/23-09.js
function isMirror(str1, str2) {

    let cleanStr1 = str1.replace(/[^a-zA-Z0-9]/g, '');
    let cleanStr2 = str2.replace(/[^a-zA-Z0-9]/g, '');
    let invertStr2 = cleanStr2.split('').reverse().join('');

    if (cleanStr1 == invertStr2) {
        
        console.log(true);
        return true;

    }else{

        console.log(false);
        return false;

    }
}

File: september/23-09/23-09.py
def is_mirror(str1, str2):

    clean_str1 = ''.join(filter(str.isalnum, str1))
    clean_str2 = ''.join(filter(str.isalnum, str2))
    clean_str2 = clean_str2[::-1]

    if(clean_str1 == clean_str2):
        return True
    else:
        return False

    

is_mirror("helloworld", "helloworld") #should return Fals
is_mirror("Hello World", "dlroW olleH") #should return True.
is_mirror("RaceCar", "raCecaR") #should return True.
is_mirror("RaceCar", "RaceCar") #should return False.
is_mirror("Mirror", "rorrim") #should return False.
is_mirror("Hello World", "dlroW-olleH") #should return True.
is_mirror("Hello World", "!dlroW !olleH") #should return True.